libs/components/packages/src/schematics/migrations/update-8/update-polyfill/update-polyfill.spec.ts

```diff
@@ -31,10 +29,9 @@ describe('updatePolyfill', () => {
       (window as any).global = window;
     `
     );
-    await runner
-      .callRule(updatePolyfillSchematic(), tree)
-      .pipe(take(1))
-      .toPromise();
+
+    await runner.runSchematic('update-polyfill', {}, tree);
```

Comment:
Running the test through `runSchematic` also validates that the `migration-collection.json` is configured correctly.